### PR TITLE
[JENKINS-32747] Adapt to Plugin POM 2.3

### PR DESCRIPTION
**Pull request by recena**
[JENKINS-32747](https://issues.jenkins-ci.org/browse/JENKINS-32747)

As part of this PR these tests should be solved:

```
hudson/scm/SubversionSCM/DescriptorImpl/url-help.jelly(org.jvnet.hudson.test.JellyTestSuiteBuilder$JellyCheck)  Time elapsed: 0.005 sec  <<< FAILURE!
java.lang.AssertionError: <?jelly escape-by-default='true'?> is missing
        at org.jvnet.hudson.test.JellyTestSuiteBuilder$JellyCheck.runTest(JellyTestSuiteBuilder.java:98)
```

@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -65,8 +67,12 @@ THE SOFTWARE.
   </distributionManagement>
 
   <properties>
-    <maven-hpi-plugin.version>1.96</maven-hpi-plugin.version>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <jenkins.version>1.568</jenkins.version>
+    <jenkins-test-harness.version>1.568</jenkins-test-harness.version>
+    <java.level>7</java.level>
```
Comment (andresrc) [inline pom.xml]: Requires Java 7 at 1.568 baseline?

File: pom.xml [hunk 2]
```diff
@@ -65,8 +67,12 @@ THE SOFTWARE.
...
 
   <properties>
-    <maven-hpi-plugin.version>1.96</maven-hpi-plugin.version>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <jenkins.version>1.568</jenkins.version>
+    <jenkins-test-harness.version>1.568</jenkins-test-harness.version>
+    <java.level>6</java.level>
+    <java.level.test>${java.level}</java.level.test>
+    <findbugs.failOnError>false</findbugs.failOnError>
+    <hpi-plugin.version>1.115</hpi-plugin.version>
```
Comment (andresrc) [inline pom.xml]: 1.115 is the current default, so no need to override, but feel free.

**Comment by andresrc** [comment]
:bee:

File: pom.xml (hunk 2 above)
Comment (recena) [inline pom.xml]: Right, default values should be removed ;) Done.

**Comment by andresrc** [comment]
re- :bee:

**Comment by stephenc** [comment]
:bee: (assuming you get the build fixed)

**Pull request merged**